stylelint.config.js

```diff
@@ -1,3 +1,9 @@
 export default {
   extends: ["stylelint-config-recommended"],
+  plugins: ["@double-great/stylelint-a11y"], // -> no `extends` https://github.com/double-great/stylelint-a11y/issues/65
+  rules: {
+    "a11y/media-prefers-reduced-motion": null,
```

Comment:
When set as `true`, this rule will result in 2 failures:

```
src/components/side-nav/side-nav.module.css
  31:5  ✖  Expected & #indicator is used with @media (prefers-reduced-motion)  a11y/media-prefers-reduced-motion

src/components/table-of-contents/table-of-contents.module.css
  41:5  ✖  Expected & #indicator is used with @media (prefers-reduced-motion)  a11y/media-prefers-reduced-motion

✖ 2 problems (2 errors, 0 warnings)
```

I wasn't sure how much you wanted to refactor those 2 instances to wrap that rule (and others as well?) in a media query to meet the rule standard. Happy to do so, but wanted to get your thoughts first.

Reply:
Yeah, this was probably an oversight on my part, we have been trying to abide by this in other components
https://github.com/ProjectEvergreen/www.greenwoodjs.dev/blob/main/src/components/capabilities/capabilities.css#L47

So we should handle these cases as well please 🙏